README.md

```diff
@@ -74,16 +78,18 @@ module BasicAutocomplete = {
                         )
                       |> Array.mapi(
                         (index, item) => {
-                          let backgroundColor = if (renderFunc##highlightedIndex === index) {
-                            "gray"
-                          } else {
-                            "white"
-                          };
-                          let fontWeight = if (renderFunc##selectedItem === item) {
-                            "bold"
-                          } else {
-                            "normal"
-                          };
+                          let backgroundColor =
+                            if (Downshift.ControllerStateAndHelpers.highlightedIndex(t) === index) {
```

Comment:
Btw you might want to use structural equality (`==`) instead of referential equality (`===`), the latter tests whether two values are pointing to the same location in memory and may not always hold true even if the values are the same.

Reply:
Oh thanks for pointing that out. I guess it’s a JS habit 😅

Do you have a link the related docs?